### PR TITLE
Add edition comparison to README.md

### PR DESCRIPTION
**Pull request by domosekai**
Since many users are not aware / confused with the difference with Stable Edition, I think a feature by feature comparison on the main page can save lives.
My list is very likely incomplete. Inputs are welcome!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +145,32 @@ https://github.com/domosekai
...
+| L2TP | ✅ | ✅ | |
+| MS-SSTP | ✅ | ✅ | |
+| L2TPv3 | ✅ | ✅ | |
+| EtherIP | ✅ | ✅ | |
+| WireGuard | ❌ | ✅ ||
+| IKEv2 | ❌ | ❌ ||
+
+| Feature | Stable | Developer | Comment |
+| --- | --- | --- | --- |
+| Password Authentication | ✅ | ✅ | |
```
Comment (metalefty) [inline README.md:164]: I think RADIUS Authentication needs to be added.

Comment (domosekai) [reply README.md:164]: Added. Thanks.

**Review by davidebeatrici — APPROVED** [review]

**Comment by davidebeatrici** [comment]
I would also specify that OpenVPN in AEAD mode is only supported in the Developer Edition.

**Comment by domosekai** [comment]
> I would also specify that OpenVPN in AEAD mode is only supported in the Developer Edition.

Thank you. I made some changes accordingly.

**Review by davidebeatrici — APPROVED** [review]

**Comment by domosekai** [comment]
Merged. Thank you. 
Feel free to add more in another PR.

**Pull request merged**